ApiSurface/Test/ApiSurface.Test.fsproj

```diff
@@ -1,6 +1,11 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup Condition="'$(CI_TEST_FRAMEWORK)'!=''">
+    <TargetFramework>$(CI_TEST_FRAMEWORK)</TargetFramework>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(CI_TEST_FRAMEWORK)'==''">
     <TargetFramework>net6.0</TargetFramework>
+  </PropertyGroup>
```

Comment:
I think the normal way to do this is to do:

```suggestion
  <PropertyGroup>
    <TargetFrameworks>net481,netcoreapp3.1,net5.0,net6.0</TargetFrameworks>
  </PropertyGroup>
```

And then in the CI, pass in an `-f net481` option to each `dotnet ...` command

Reply:
I did try that - commit 6a9667abe8dc7f54e33dcd36ef4b1e55971f8fc5 and its parent 9d5f4120b0777b97d6e64ecd0ed213141fe86aa3. The result was e.g.

> Error: /usr/share/dotnet/sdk/6.0.402/Sdks/Microsoft.NET.Sdk/targets/Microsoft.PackageDependencyResolution.targets(267,5): error NETSDK1005: Assets file '/home/runner/work/ApiSurface/ApiSurface/ApiSurface.DocumentationSample/obj/project.assets.json' doesn't have a target for 'netstandard2.0'. Ensure that restore has run and that you have included 'netstandard2.0' in the TargetFrameworks for your project. [/home/runner/work/ApiSurface/ApiSurface/ApiSurface.DocumentationSample/ApiSurface.DocumentationSample.fsproj]

I'd have to restore as we do on master (so as to get netstandard restores on ApiSurface), but then do another restore on the test project with the different framework, right?

Reply:
Unless you mean have ApiSurface itself multi-target?

Reply:
This will greatly bloat the NuGet package, and seems a bit sad to me given that netstandard2.0 does already cover everything.

Reply:
OK, as long as we don't touch the restore stage, it does Just Work! I take it all back